core/arch/x86/decode_private.h

```diff
@@ -149,7 +149,7 @@ enum {
  *                     if bit 3 (OPCODE_MODRM) set, opcode based on entire modrm
```

Comment:
Please use a more specific title than "Add more EVEX instructions" which seems like it could apply to the next 10 commits such that we can't tell them apart.

Reply:
Done.